src/mobile/dropdown/index.ts

```diff
@@ -0,0 +1,268 @@
+      return;
+    }
+    this.className =
+      props.className !== undefined ? props.className : this.className;
+    this.error = props.error !== undefined ? props.error : this.error;
+    this.id = props.id !== undefined ? props.id : this.id;
+    this.label = props.label !== undefined ? props.label : this.label;
+    this.value = props.value !== undefined ? props.value : this.value;
+    this.disabled =
+      props.disabled !== undefined ? props.disabled : this.disabled;
+    this.requiredIcon =
+      props.requiredIcon !== undefined ? props.requiredIcon : this.requiredIcon;
+    this.visible = props.visible !== undefined ? props.visible : this.visible;
+    this.items = props.items !== undefined ? props.items : this.items;
+  }
+
+  private _generateGUID(): string {
+    return (
+      new Date().getTime().toString(16) +
+      Math.floor(Math.random() * 0x1000).toString(16)
+    );
+  }
+
+  private _updateVisible() {
+    if (!this.visible) {
+      this.setAttribute("hidden", "");
+    } else {
+      this.removeAttribute("hidden");
+    }
+  }
+
+  private _handleChangeInput(event: Event) {
+    event.stopPropagation();
+    const selectEl = event.target as HTMLSelectElement;
+    const detail: CustomEventDetail = { value: "", oldValue: this.value };
+    this.value = selectEl.value;
+    detail.value = this.value;
+    this._dispatchCustomEvent("change", detail);
+  }
+
+  private _dispatchCustomEvent(eventName: string, detail?: CustomEventDetail) {
+    const event = new CustomEvent(eventName, {
+      detail,
+      bubbles: true,
+      composed: true
+    });
+    this.dispatchEvent(event);
+  }
+
+  createRenderRoot() {
+    return this;
+  }
+
+  update(changedProperties: PropertyValues) {
+    if (changedProperties.has("items")) {
+      this._validateItems();
+    }
+    super.update(changedProperties);
+  }
+
+  private _getItemTemplate(item: Item, index: number) {
+    return html`
+      <option
+        value="${item.value || ""}"
+        ?selected="${this.value === item.value}"
+      >
+        ${item.label === undefined ? item.value : item.label}
+      </option>
+    `;
+  }
+
+  render() {
+    this._updateVisible();
+    return html`
+      ${this._getStyleTagTemplate()}
+      <label
+        class="kuc-mobile-dropdown__label"
+        for="${this._GUID}-label"
+        ?hidden="${!this.label}"
+      >
+        <span class="kuc-mobile-dropdown__label__text">${this.label}</span
+        ><!--
+        --><span
+          class="kuc-mobile-dropdown__label__required-icon"
+          ?hidden="${!this.requiredIcon}"
+          >*</span
+        >
+      </label>
+      <div class="kuc-mobile-dropdown__input-form">
+        <div class="kuc-mobile-dropdown__input-form__select">
+          <select
+            class="kuc-mobile-dropdown__input-form__select__input"
+            id="${this._GUID}-label"
+            aria-describedBy="${this._GUID}-error"
+            aria-required="true"
+            aria-invalid="true"
```

Comment:
The "aria-required" and "aria-invalid" have condition to set "true". So, could you please update code following spec?